index.js

```diff
@@ -0,0 +1,232 @@
+import { projects, menu, footerLinks } from '/constants.js';
+
+let items = [];
```

Comment:
You are already familiar with css modules. Please consider splitting this file into multiple ones